src/payment.gql

```diff
@@ -0,0 +1,139 @@
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.3", import: ["@composeDirective", "@extends", "@external", "@inaccessible", "@interfaceObject", "@key", "@override", "@provides", "@requires", "@shareable", "@tag"])
+
+"""A payment of an invoice or return"""
+type Payment
+  @key(fields: "id")
+{
+  """The uuid identifier of the product item"""
+  id: UUID!
+
+  """Payment Amount in EUR"""
+  amount: Float!
+
+  """Date of the payment"""
+  payedAt: DateTime!
+
+  """Status of the payment"""
+  status: PaymentStatus!
+}
+
+"""A universally unique identifier compliant UUID Scalar"""
+scalar UUID
+
+"""
+A date-time string at UTC, such as 2019-12-03T09:54:33Z, compliant with the date-time format.
+"""
+scalar DateTime
+
+"""The status of an payment of an invoice or return"""
+enum PaymentStatus {
+  """The payment was created but not yet processed"""
+  OPEN
+
+  """The payment is currently being processed"""
+  PROCCESSING
+
+  """The payment was successfully processed and the amount was transfered"""
+  SUCCEEDED
+
+  """The payment processing failed indefinetely"""
+  FAILED
+
+  """The payment was sold to external inkasso service"""
+  INKASSO
```

Comment:
I think this this is not an English term (at least google translate says it's Norwegian)

Reply:
I will keep this in mind for the /payment PR